Vagrantfile.example

```diff
@@ -35,6 +25,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     }
  end
 
+ # Comment this line if you would like to disable the automatic update during provisioning
+ config.vm.provision "shell", inline: "sudo dnf upgrade -y"
+
+ # bootstrap and run with ansible
+ config.vm.provision "shell", path: "playpen/vagrant-bootstrap-ansible.sh"
```

Comment:
Fun fact: That was its original name, before I knew the truth of how this all worked together. So yes. :)